vhdbuilder/packer/install-dependencies.sh

```diff
@@ -395,11 +396,96 @@ for KUBE_PROXY_IMAGE_VERSION in ${KUBE_PROXY_IMAGE_VERSIONS}; do
   CONTAINER_IMAGE="mcr.microsoft.com/oss/kubernetes/kube-proxy:v${KUBE_PROXY_IMAGE_VERSION}"
   pullContainerImage ${cliTool} ${CONTAINER_IMAGE}
   ctr --namespace k8s.io run --rm ${CONTAINER_IMAGE} checkTask /bin/sh -c "iptables --version" | grep -v nf_tables && echo "kube-proxy contains no nf_tables"
-  
+
   # shellcheck disable=SC2181
   echo "  - ${CONTAINER_IMAGE}" >>${VHD_LOGS_FILEPATH}
 done
 
+# download and setup azcopy to use to download private packages with MSI auth
+getAzCopyCurrentPath() {
```

Comment:
probably better to move these funcs to a separate file, such as vhdbuilder/packer/scripts/tools_installs.sh

Reply:
done, thanks